docs/can-guides/commitment/recipes/todomvc-with-steal/todomvc-with-steal.md

```diff
@@ -231,23 +231,23 @@ QUnit.equal(todo.complete, true, "toggleComplete works");
 - The [can-define.types.type type] behavior defines a property’s type like:
 
   ```js
-  DefineMap.extend({
+  const Map = DefineMap.extend({
```

Comment:
Generally I would avoid `Map` because it’s a built-in constructor.

Reply:
Ah good point. Will rename